### ISSUE TITLE
Unable to Create Post on dev.to - Status 500 Internal Server Error

### ISSUE THREAD

**Issue by engrsakib** [forem/forem#21338]
Describe the bug

I am unable to create a post on dev.to. Every time I attempt to submit a post, I receive a "Whoops, something went wrong: Status: 500 Internal Server Error."

To Reproduce

Go to dev.to and log in to my account.
Click on "Create Post."
Fill in the post details.
Click on "Publish."
See the error message.
Expected behavior

I expected the post to be published successfully without any error messages.

Screenshots
![image](https://github.com/user-attachments/assets/7bfa10e9-416a-4d50-8af7-32518781d15b)

Desktop:

OS, version: Windows 10
Browser, version: Microsoft Edge, version 115.0.1901.188
Browser, version: Google Chrome, latest version

Device: Techno Spark 10 Pro
OS, version: Android 13
Browser, version: Chrome, version 117.0.5938.90
Additional context

I have tried accessing the site using different browsers (Edge, Chrome) and devices, but the issue persists. I also disabled all browser extensions to see if that resolved the problem, but it did not. I am located in Bangladesh, and it seems that others in the region are able to post without issues.

**Comment by traez** [comment]
Yes, it's been happening a lot today, off and on. The team needs to check their server-side code.

They use amazonaws for file uploads, and they didn't even bother to mask it. Lols. We're all learning.
